src/Properties/ModelRelationsExtension.php

```diff
@@ -44,11 +45,19 @@ public function hasProperty(ClassReflection $classReflection, string $propertyNa
             return false;
         }
 
+        if (Str::endsWith($propertyName, '_count')) {
+            $propertyName = Str::beforeLast($propertyName, '_count');
+        }
+
         return $classReflection->hasNativeMethod($propertyName);
     }
 
     public function getProperty(ClassReflection $classReflection, string $propertyName): PropertyReflection
     {
+        if (Str::endsWith($propertyName, '_count')) {
+            return new ModelProperty($classReflection, IntegerRangeType::fromInterval(0, null), IntegerRangeType::fromInterval(0, null), false);
```

Comment:
This property should not be writable I guess. So you can give `NeverType` as writable type.